lib/open_food_network/variant_stock.rb

```diff
@@ -22,39 +34,65 @@ def on_hand
       end
     end
 
+    # Alias for Spree's 2.0 #total_on_hand
```

Comment:
Instead of referring to the method used below. It would be more useful to explain what this method does, probably just "Number of items available in the stock for this variant."